dnf5/main.cpp

```diff
@@ -1493,6 +1494,13 @@ int main(int argc, char * argv[]) try {
                                 "checks will be performed."));
                         }
                     }
+                    if (libdnf5::utils::bootc::is_bootc_system()) {
```

Comment:
This does not take into account --downloadonly option as DNF4 does. DNF4 users complained that they want to download packages (into standard /var/cache path) even on a locked bootc system. Probably they use it for mirroring updates.

Reply:
I think it does respect `downloadonly` (and `--store` too) since this logic is in the `else` block here, right?

Reply:
You are right. I overlooked the nesting.